### ISSUE TITLE
Add "New Content" to the Content Changed Digest

### ISSUE THREAD

**Issue by wgilling** [asulibraries/islandora-repo#235]
This should be a little simpler than the changed content section of the report. The new view would simply be filtered for asu_repository_items that have a created date that is restricted to the report date. This view may be able to be added as pages to the existing Differences view or it may even be possible that the "Recent Items" view (machine id "content_recent").

The same code can be used to pull in both the "Differences" view and the "New Content" view. It is possible that the same twig can be used to style the output of either section.

**Comment by wgilling** [comment]
I wonder if this should not include unpublished nodes as long as the email makes it clear that those items are not published yet... because if the state changes from Draft to Published the node will NO LONGER appear on the "New Content" report but since there would be a revision now, it would be included with the "Content Changed 1 day" report view.

**Comment by elizoller** [comment]
are you saying that the new content would or wouldn't include unpublished nodes? are you concerned about duplicates in the two sections: new and updated? maybe updated should not include nodes which were created on that day

**Comment by wgilling** [comment]
I can have the filter set to "Published status or admin user" and the code that goes through this execution as CRON should be administrator but the "Differences view" is set to "Content revision: Published (= Yes)"... so do we need to think about this? 

Also, should there be a limit on how many to display only 10 or 20 new items (in cases where 100+ have been migrated in a single day, it'd be a big email) - and render a link to the view path /newcontent/1day at the bottom of the email?

**Comment by wgilling** [comment]
the theming seems to work in the latest branch of the code after I did a full config:import. I suspect that the view was cached somehow and was using the old machine name internally or something like that.

All that I did before testing the emailing was:
```
git checkout newcontent_digest_235
git pull
cd ..
drupal config:import --directory /var/www/html/drupal/config/sync
cd web/
drush cr
drush updatedb
```

Then, I configured SMTP and made sure that I was on the list to get the emailing when I ran CRON.

**Comment by wgilling** [comment]
It would be nice to add a message in the event that there are no nodes returned for the differences/1day and newcontent/1day such as "there is no new content in the last day"

**Comment by wgilling** [comment]
When there is no content for the differences/1day or newcontent/1day views, the output of the email will look like this now.

![Screen Shot 2020-12-17 at 2 24 53 PM](https://user-images.githubusercontent.com/19391126/102545603-b5c5fc00-4073-11eb-81a5-f88fe64ab127.png)

**Comment by elizoller** [comment]
closed with #255
